packages/deployments/contracts/src/cli/init/config/testnet/production.ts

```diff
@@ -279,6 +279,26 @@ export const TESTNET_PRODUCTION_INIT_CONFIG: InitConfig = {
         },
       },
     },
+    {
+      name: "InrETH",
+      canonical: {
+        domain: "1735353714",
```

Comment:
change this to pseudo domain. 
Example:
https://github.com/connext/monorepo/blob/dc6a1fd74bf5a1448bdb1cef9006348d60e47031/packages/deployments/contracts/src/cli/init/config/testnet/production.ts#L262